isb_web/manage.py

```diff
@@ -207,6 +209,8 @@ async def login(request: starlette.requests.Request):
     # check if login is for annotation purpose, if so add query param
     if "annotation" in request.query_params and request.query_params["annotation"] == "true":
         redirect_uri += "?annotation=true"
+    elif "thing" in request.query_params:
+        redirect_uri += f"?thing={request.query_params['thing']}"
```

Comment:
New support to bounce back over to the thing page when redirecting back from orcid login.